tests/cypress/support/commands.ts

```diff
@@ -22,9 +22,10 @@ import * as cypressLib from '@rancher-ecp-qa/cypress-library';
 // Fleet commands
 
 // Add path on "Git Repo:Create"
-Cypress.Commands.add('addPathOnGitRepoCreate', (path) => {
+Cypress.Commands.add('addPathOnGitRepoCreate', (path, index=0) => {
+  //Index defaulting to 0, for first input box.
   cy.clickButton('Add Path');
-  cy.get('input[placeholder="e.g. /directory/in/your/repo"]').type(path);
+  cy.get(`[data-testid="array-list-box${ index }"] input[placeholder="e.g. /directory/in/your/repo"]`).type(path);
```

Comment:
Pls confirm this works in 2.7 and 2.9.

Reply:
CI 2.7 Run link: https://github.com/rancher/fleet-e2e/actions/runs/10175819361/job/28143987741
<details>
<summary> Screenshot showing test passed. </summary>

![image](https://github.com/user-attachments/assets/406dc3e5-a028-4e34-9bc5-ea2e9231cdc4)

</details>

CI 2.9 Run link: https://github.com/rancher/fleet-e2e/actions/runs/10175822635

Reply:
Latest CI 2.7 run: https://github.com/rancher/fleet-e2e/actions/runs/10176772477

Reply:
Latest CI 2.9 run: https://github.com/rancher/fleet-e2e/actions/runs/10177343063